src/plots/gl2d/index.js

```diff
@@ -27,6 +30,21 @@ exports.attrRegex = constants.attrRegex;
 
 exports.attributes = require('../cartesian/attributes');
 
+// gl2d uses svg axis attributes verbatim, but overrides editType
+// this could potentially be just `layoutAttributes` but it would
+// still need special handling somewhere to give it precedence over
+// the svg version when both are in use on one plot
+exports.layoutAttrOverrides = overrideAll(Cartesian.layoutAttributes, 'doplot', true);
+
+// similar overrides for base plot attributes (and those added by components)
+exports.baseLayoutAttrOverrides = overrideAll({
+    plot_bgcolor: Plots.layoutAttributes.plot_bgcolor,
```

Comment:
I think this too could be :hocho: after #1869 where gl2d subplot with reused svg axis layering.